libbeat/scripts/Makefile

```diff
@@ -206,18 +206,6 @@ coverage-report:
 update:
 	bash ${ES_BEATS}/libbeat/scripts/update.sh ${BEATNAME} ${BEAT_DIR}/${BEATNAME} ${ES_BEATS}/libbeat
 
-### KIBANA FILES HANDLING ###
-
-KIBANA_ES?=http://localhost:9200/
-.PHONY: update
-kibana-export:
```

Comment:
The idea to have it here was to make it easily available for community beats developers.